src/docs/devices/Brilliant-Smart-20696-CCT-light/index.md

```diff
@@ -33,10 +33,11 @@ substitutions:
 esphome:
   name: $device_name
   comment: $device_description
-  platform: ESP8266
-  board: esp01_1m
   esp8266_restore_from_flash: true
```

Comment:
This PR should also include moving `esp8266_restore_from_flash` from the `esphome` block to `restore_from_flash` inside the new `esp8266` block